src/ui/PublicationCard/PublicationCard.jsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 
 function PublicationCard({ children, ...rest }) {
-  return <div className="eea publication card">{children}</div>;
+  return (
+    <div className="eea publication card">
+      <a href={rest.href}>{children}</a>
```

Comment:
The link here shouldn't be here. Leave the link wrapping to the integration code. For example, an integrator may do:
```
<PublicationCard>
<a href=""><PublicationCard.Info>dasdasdas</PublicationCard></a>
</PublicationCard>
```
or they may do:

```
<PublicationCard>
<Modal target={<PublicationCard.Info ..../>} />
</PublicationCard>
```

Reply:
This component is still in progress. It will be adjusted in next versions.